plans/terraform/destroy.pp

```diff
@@ -19,9 +19,12 @@
 plan provision::terraform::destroy(
   String[1]                    $tf_dir,
   String[1]                    $provider,
-  String[1]                    $resource_name = undef,
+  String[1]                    $resource_name,
   Optional[String[1]]          $region        = undef,
-  Optional[Hash[String[1], String[1]]] $provider_options = undef,
+  Optional[Provision::ProviderOptions] $provider_options = undef,
+  String[1]                    $profile            = undef,
```

Comment:
Not sure what is meant here: is `$profile` a required or an optional parameter? If required, why does it have a default value of undef in the first place?

Reply:
Yeah, I feel its not needed here as well. Thanks for pointing out. Made the required change in the PR.